crates/networking/p2p/src/constants.rs

```diff
@@ -4,3 +4,10 @@ use alloy_primitives::{aliases::B32, fixed_bytes};
 pub const MAX_PAYLOAD_SIZE: u64 = 10485760;
 pub const MESSAGE_DOMAIN_VALID_SNAPPY: B32 = fixed_bytes!("0x01000000");
 pub const MESSAGE_DOMAIN_INVALID_SNAPPY: B32 = fixed_bytes!("0x00000000");
+
+pub const STATE_CONNECTED: &str = "connected";
+pub const STATE_CONNECTING: &str = "connecting";
+pub const STATE_DISCONNECTED: &str = "disconnected";
+pub const STATE_DISCONNECTING: &str = "disconnecting";
+pub const DIR_INBOUND: &str = "inbound";
+pub const DIR_OUTBOUND: &str = "outbound";
```

Comment:
These constants should be in the p2p crate, because they are only used in that crate

also please avoid shorthand in `DIR_INBOUND` for reference

Reply:
Copy that. I’ve removed these constants since I’ll be using enums for the state and direction fields in the CachedPeer struct.